src/theme/theme.ts

```diff
@@ -11,6 +11,13 @@ export const getDesignTokens = (mode: PaletteMode): ThemeOptions => ({
           text: {
             primary: "#252733",
           },
+          score: {
+            red: "#FAC8D0",
```

Comment:
- [ ] declare variables for colors and update them in all files where you use colors

Reply:
done